testbench_performance.py

```diff
@@ -20,49 +20,19 @@
   $ py.test -vs testbench_performance.py --benchmark-save=bench1
 """
 
-import atexit
-import json
 import logging
-import os
-import shutil
 import pytest
-import tempfile as tmp
 
 import minibench
 
-# TODO: Yank these two from the config options.
-WORKSPACE = tmp.mkdtemp()
-# param_file = pytest.config.getoption("--param_file")
-DATA_PARAMS = json.load(open("./params.json"))
 
 # TODO: Add verbosity to the config options.
 logging.basicConfig(level=logging.INFO)
 
-logging.info("Using {} for workspace".format(WORKSPACE))
-
-
-@pytest.fixture(params=DATA_PARAMS,
-                ids=["{}".format(p) for p in DATA_PARAMS],
-                scope="module")
-def npy_files(request):
-    """Populate a temporary stash file with data."""
-    fset = minibench.data.create_npy_collection(
-        shape=request.param['shape'],
-        num_items=request.param['num_items'],
-        output_dir=WORKSPACE)
-    return fset
+# TODO: Can we still do this with fixtures?
+# logging.info("Using {} for workspace".format(workspace))
```

Comment:
I'm not sure which way you want it... but if you want a new one for each, you could just get rid of scope="module"